packages/taquito-rpc/src/types.ts

```diff
@@ -11,6 +11,64 @@ export type DelegateResponse = string | null;
 
 export type OperationHash = string;
 
+interface Before {
+  value?: string;
+  node?: string;
+}
+
+interface After {
+  value?: string;
+  node?: string;
+}
+
+interface State {
+  inode?: Inode;
+  other_elts?: any;
```

Comment:
other_elts can have a proper type instead of any

Reply:
fixed the type